docs/language/src/builtins/widgets.md

```diff
@@ -462,6 +462,36 @@ export component Example inherits Window {
 }
 ```
 
+## `Switch`
+
+A `Switch` is used to toggle between `checked` and `unchecked` state represented by a boolean.
```

Comment:
Perhaps here it would be good to have some guidance how this differs from a `CheckBox`. And in the `CheckBox` docs some guidance how it differs from a `Switch`.

I quite like the form example you told me about the other day, but just about any simple example would do :). If you're not sure, then I can also try to come up with some text.

Reply:
I try something ;-), you can check and update it if you like.